### ISSUE TITLE
Source(s) for the  formats

### ISSUE THREAD

**Issue by oneEyedSunday** [agbowo/sms-parser#2]
Hello, do you have source(s) for the formats?
or its a trial and error thing?

How do you know when they change?

Or variations based on say `version of sms` or `bank account type` of customer?

**Comment by OAyomide** [comment]
yeah its trial and error. 

i take an example of sms from a bank and parse it. The formats are just text (sms text). a bank can also send promotional texts or texts that are not alerts. in the next iteration, i'd first scan for a keyword like cr or dr and then parse or ignore accordingly

**Comment by oneEyedSunday** [comment]
Nice. I guess you could try to source for more smses. To make it more fool
proof
There's a bunch of them on twitter.
Albeit, I'd take anything finance related on twitter with a pinch of salt

**Comment by OAyomide** [comment]
Yeah thats the idea. work still has to be done, corner cases and more banks to be parsed.
